servo/connectors/kubernetes.py

```diff
@@ -875,10 +876,18 @@ async def create(self, namespace: str = None) -> None:
         if namespace is None:
             namespace = self.namespace
 
-        self.logger.info(f'creating pod "{self.name}" in namespace "{self.namespace}"')
+        self.logger.info(f'Attempting to create a pod named {self.name} in the namespace {self.namespace}.')
         self.logger.trace(f"pod: {self.obj}")
 
         async with self.preferred_client() as api_client:
+            async with kubernetes_asyncio.watch.Watch().stream(api_client.list_pod_for_all_namespaces) as stream:
+                async for event in stream:
+                    event_type, event_object = event['type'], event['object']
+                    # TODO: Really should find a way to select in the API call so that we aren't registering for all events for all time.
+                    # This will be frowned upon in permissive environments where our API calls will return many more pods.
+                    if event_object.metadata.namespace == self.namespace:
+                        self.logger.info(f'Kubernetes reports that we successfully {event_type.lower()} a pod named {event_object.metadata.name} in the {event_object.metadata.namespace} namespace.')
```

Comment:
For pretty formatting purposes I've gone ahead and done some string manipulation, but every part of me is saying, "You can't be guaranteed that Kubernetes API call results are strings. What are you even thinking? That's angrily coercing data types in a way that isn't even close to explicit." And yet another part of me goes, "I bet that's just the Python way of doing things."

Thoughts?

Reply:
Watch event's `type` field is a [str in the backing model](https://github.com/kubernetes-client/python/blob/master/kubernetes/docs/V1WatchEvent.md), so it should be guaranteed to be a string by the protocol buffers